apps/molgenis-components/src/client/client.ts

```diff
@@ -307,6 +307,7 @@ const fetchOntologyOptions = async (
   const tableDataQuery = `query ${tableId} {
         ${tableId}(
           limit:100000
+          orderby: { name: ASC }
```

Comment:
I thought the issue is that data managers can set the 'order' column and that that should be used? Or is now decided that alphabetical order is what is desired?